superset-frontend/plugins/plugin-chart-echarts/test/Timeseries/transformProps.test.ts

```diff
@@ -82,19 +86,20 @@ describe('EchartsTimeseries tranformProps', () => {
   it('should add a formula annotation to viz', () => {
     const formula: FormulaAnnotationLayer = {
       name: 'My Formula',
-      annotationType: 'FORMULA',
+      annotationType: AnnotationType.Formula,
       value: 'x+1',
-      style: 'solid',
+      style: AnnotationStyle.Solid,
+      showLabel: true,
       show: true,
     };
-    const chartProps = new ChartProps({
+    const chartProps: unknown = new ChartProps({
```

Comment:
Same question as above, would any of [these prop types](https://github.com/apache/superset/blob/6f8927209b80dc61ba14fc66ff94871b36a83321/superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts#L32-L37) work instead of `unknown` here?